http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -138,7 +139,9 @@ class LeoPubsubMessageSubscriber[F[_]](
   private[monitor] def messageHandler(event: Event[LeoPubsubMessage]): F[Unit] = {
     val traceId = event.traceId.getOrElse(TraceId("None"))
     val now = Instant.ofEpochMilli(com.google.protobuf.util.Timestamps.toMillis(event.publishedTime))
-    implicit val appContext = Ask.const[F, AppContext](AppContext(traceId, now))
+    val span = Tracing.getTracer.spanBuilder("leoPubsubMessage").startSpan()
```

Comment:
Create a `Span` for the lifecycle of pubsub message handling. For now just creates 1 parent span, no children (will experiment though).